test/verify/check-accounts

```diff
@@ -100,8 +100,11 @@ class TestAccounts(MachineCase):
         b.wait_in_text('#accounts-list', "Berta Bestimmt")
 
         # Check home directory
-        self.assertIn(":/home/berta:", m.execute("getent passwd berta"))
-        self.assertEqual(m.execute("stat -c '%U' /home/berta").strip(), "berta")
+        home_dir = "/home/berta"
+        if m.atomic_image:
+            home_dir = "/var" + home_dir
```

Comment:
This isn't true for the other atomics (rhel/continuous). Let's just relax this a bit by actually parsing the home directory,  like `getent passwd berta | cut -f6 -d:`, and then stat'ing to make sure it exists. The point of this was to make sure that  we actually create the home directory, but it's not actually our worry where exactly it is.